### DISCUSSION TITLE
Facebook videos doesn't play due to FX_cast

### DISCUSSION THREAD

**Discussion by jonnisell** [hensm/fx_cast#168] [Q&A] [answered]
Hi community,
Facebook videos doesn't play for me and I've isolated the issue to FX_cast add-on in Firefox. I opened up Developer tools, but nothing seems wrong or no data missing in fetching a page. 
The issue is hence, I'm not trying to cast anything and afaik Facebook's player doesn't enable streaming, yet the plugin blocks videos playing on regular Facebook pages.

My env:
Mac OSX 10.15.7
FF version 85.0.2
Plugin v0.1.2 and bridge v0.1.0

**Comment by hensm** [comment, marked as answer]
Can confirm it breaks something. Just remove facebook from the whitelist for now.
